cw_core/lib/node.dart

```diff
@@ -134,6 +136,7 @@ class Node extends HiveObject with Keyable {
     try {
       switch (type) {
         case WalletType.monero:
+        case WalletType.wownero:
           return requestMoneroNode();
         case WalletType.bitcoin:
           return requestElectrumServer();
```

Comment:
```suggestion
         case WalletType.monero:
         case WalletType.wownero:
         case WalletType.haven:
           return requestMoneroNode();
         case WalletType.bitcoin:
           return requestElectrumServer();
```

perhaps ^ + del L145-L146 

(missing a space / tab is misaligned for the wownero case line)